### PR TITLE
Fixed links to docs

### PR DESCRIPTION
**Pull request by goposky**
Fixed links to docs

### CODE REVIEW

File: website/content/en/blog/2024-03-15-green-reviews-working-group-moving-towards-measuring-the-sustainability-footprint-of-cncf-projects.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The continuous operation of a Kubernetes cluster raises questions about how sust
...
 
 Please keep in mind that only some components of the described vision are in production at the time of writing this blog post. If you would like to find out more about the latest status of the work that’s being done in the Green Reviews WG, please join our community in the [Green Reviews WG Slack channel](https://cloud-native.slack.com/archives/C060EDHN431) or the [TAG ENV Slack channel](https://cloud-native.slack.com/archives/C03F270PDU6), or check out the [documentation](https://github.com/cncf-tags/green-reviews-tooling/blob/main/website/content/docs) on GitHub.
```
Comment (guidemetothemoon) [inline website/content/en/blog/2024-03-15-green-reviews-working-group-moving-towards-measuring-the-sustainability-footprint-of-cncf-projects.md:75]:
```suggestion
Please keep in mind that only some components of the described vision are in production at the time of writing this blog post. If you would like to find out more about the latest status of the work that’s being done in the Green Reviews WG, please join our community in the [Green Reviews WG Slack channel](https://cloud-native.slack.com/archives/C060EDHN431) or the [TAG ENV Slack channel](https://cloud-native.slack.com/archives/C03F270PDU6), or check out the [documentation](https://github.com/cncf-tags/green-reviews-tooling/blob/main/docs) on GitHub.
```

**Review by guidemetothemoon — COMMENTED** [review]
@goposky thank you for fixing this! I've detected two more places where the link is outdated and I added a change suggestion for one but couldn't directly add it for the other place which is on line 107 - can you please update those two links as well?
Thanks!🙌

**Comment by goposky** [comment]
> @goposky thank you for fixing this! I've detected two more places where the link is outdated and I added a change suggestion for one but couldn't directly add it for the other place which is on line 107 - can you please update those two links as well? Thanks!🙌

@guidemetothemoon done

**Review by guidemetothemoon — APPROVED** [review]
LGTM, thanks!

**Pull request merged**